cyber/common/file.cc

```diff
@@ -123,10 +123,11 @@ bool GetContent(const std::string &file_name, std::string *content) {
 }
 
 std::string GetAbsolutePath(const std::string &prefix,
-                            const std::string &relative_path) {
+                            const std::string &relative_path) const {
```

Comment:
Thanks for your contribution!
GetAbsolutePath is not a member function of a class, so it is unnecessary to mark it as "const function".
If you want mark return type as const, you should put 'const' at the beginning of the function.

Reply:
Thanks for your comments! well, that's wrong of me to saw it a class member somehow. will revise it since am intended to have it return as const.